src/screens/TopicMemberScreen/index.js

```diff
@@ -150,13 +174,14 @@ const TopicMemberScreen = () => {
 
   const handleScrollEvent = React.useCallback(
     (event) => {
+      Keyboard.dismiss();
       const {y} = event.nativeEvent.contentOffset;
       if (y <= 15) {
         showAnimationHeader();
       } else {
         interactionManagerAnimatedRef.current = InteractionManager.runAfterInteractions(() => {
           Animated.timing(animatedHeight, {
-            toValue: dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT2,
+            toValue: dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT2 + topPosition,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Use the helper function `calculateAnimatedHeight` here as well.
```diff
-  Animated.timing(animatedHeight, {
-   toValue: dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT2 + topPosition,
-   duration: 100,
-   useNativeDriver: false
- }).start();
+ Animated.timing(animatedHeight, {
+   toValue: calculateAnimatedHeight(),
+   duration: 100,
+   useNativeDriver: false
+ }).start();
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->